projectcharter.md

```diff
@@ -0,0 +1,84 @@
+# forms.gov Project Charter
+Constituents of the Federal government use some number of more than 23,000 different forms available to them to interact with agencies; these agencies use the information from these forms to provide information or benefits, but spend innumberable hours recreating and quality checking handwritten entries on the original paper forms. This method of manual form processing based on paper-only forms leads to more than 11.4 billion hours of paperwork annually. As we have seen in multiple USDS engagements, web-based forms created through user-centered design methods reduce the respondent burden while producing well-structured data to the agencies for ingestion. This project creates a foundation for optimizing the process of building more useful, modern, and frictionless web-based forms.
+  
+  
+## Stakeholders
+#### Who needs to be informed about progress in major decisions?
+* Sponsor - Suzette King (OFCIO)
+* USDS Leadership
+* DHS Digital Service team
+* Others?
+
+## Project communication expectations
+**What is the best method for staying in touch?**
+* Hipchat - Paperless.gov channel
+* GitHub - us-forms-system repo
+* Email
+* Others?
+
+## What should status reports include?
+Progress towards project goals, blockers
+
+## Key Players
+* Julie Meloni | Project Lead
+* Ellen Butters | UX Designer
+* Anne Krainic | Engineer
+* Ju-Lie McReynolds | Product Manager
+* Others?
+
+## Project goal
+#### What will be the end result of the project?  
+A foundation for optimizing the process of building more useful, modern, and frictionless web-based forms through the creation of a public open-source forms library that will incorporate the U.S. Web Design Standards and include guidance and best practices for use.
+
+## Scope
+#### What is in scope?
+1. The aggregation and public sharing of lessons-learned from multiple years of USDS projects involving the transformation of forms from paper to web-based.
+2. The creation and public open-source release of a forms library based on the DSVA extensions to RSJF. This library could be used government-wide, and will incorporate the U.S. Web Design Standards along with guidance for data storage and integration best practices.
+3. The use of this library in production by at least one agency (besides VA).
+4. The creation of exemplars for conversion of high-value paper-based forms to web-based forms using this library.
+
+#### What is not in scope?
+* Support for additional form builders or rendering non-React components
+* Integration with other government digital services (e.g. login.gov)
+* Others?
```

Comment:
Do we want to list features that will not be supported in the first release? Such as saving a form in progress, prefilling existing information, back-end connections to internal services, non-linear form progress (this is one we haven't fully discussed yet).